lua/cyberdream/extensions/telescope.lua

```diff
@@ -42,7 +42,7 @@ function M.get(opts, t)
         end
         highlights.TelescopeResultsBorder = { fg = t.bgAlt, bg = t.bgAlt }
         highlights.TelescopeResultsNormal = { bg = t.bgAlt }
-        highlights.TelescopeResultsTitle = { fg = t.bgAlt, bg = t.bgAlt }
+        highlights.TelescopeResultsTitle = { fg = t.bgAlt, bg = t.cyan }
```

Comment:
Can we flip the bg and fg colours for this group? Just so it looks a bit closer to the screenshots in the readme 🙂 

```suggestion
        highlights.TelescopeResultsTitle = { fg = t.cyan, bg = t.bgAlt }
```

Reply:
Hmmm, now that I look at your screenshots in #124, it makes sense with these highlights because it's right at the top of the window. But with my Telescope config, it looks a bit odd:

![image](https://github.com/user-attachments/assets/3699699f-4a7f-4928-93db-54d3d7023146)

This is due to my `prompt_position` setting in my telescope config which is not default. So I say we leave it as is. I'll configure this as a separate telescope style or an override instead.